pkg/model/manifest_friendly_err.go

```diff
@@ -46,7 +47,7 @@ func getManifestSuggestionRules(manifestSchema interface{}) []*suggest.Rule {
 
 	for structName, structKeywords := range manifestKeys {
 		for _, keyword := range structKeywords {
-			// example: line 5: field contex not found in type model.buildInfoRaw
+			// example: line 5: field context not found in type model.buildInfoRaw
```

Comment:
```suggestion
			// example: line 5: field contex not found in type model.buildInfoRaw
```
I think `contex` was the example of a invalid key